beps/0003-auth-architecture-evolution/README.md

```diff
@@ -232,22 +244,21 @@ export type BackstageUnauthorizedCredentials = {
 
 type BackstageCredentialTypes = {
   user: BackstageUserCredentials;
+  'user-cookie': BackstageUserCredentials;
   service: BackstageServiceCredentials;
   unauthorized: BackstageUnauthorizedCredentials;
 };
 
 export interface HttpAuthService {
-  createHttpPluginRouterMiddleware(options: OptionsTBD): Handler;
-
+  // Implementations should cache resolved credentials on the request object
```

Comment:
👀 realized we can remove `createHttpPluginRouterMiddleware` - details in PR body